tests/unit/models/360-core.js

```diff
@@ -1,6 +1,6 @@
 describe("SerSol 360 Core Model >", function() {
   var serSol360Core = {}, journalResponse = {}, scope = {}, titles = [], searchResults = {};
-  var results = "<div ui-view='searchResults'><div class='results-title-data'><div class='results-title-row'><div class='results-title-image-div'><img src='' ng-src='' class='results-title-image'/></div><div class='results-title-details'><div class='results-title'>The New England journal of medicine</div><div class='results-identifier'>ISSN: 0028-4793</div></div></div></div></div>";
+  var results = "<div ui-view='searchResults'><div class='results-title-data'><div class='results-title-row'><div class='results-title-image-div'><img src='' ng-src='' class='results-title-image'/></div><div class='results-title-details'><div class='results-title'>The Base Journal</div><div class='results-identifier'>ISSN: 0028-4793</div></div></div></div></div>";
```

Comment:
_non-blocking PR comment_

When I saw this change, I thought, "oh, okay, base journal must be in the test somewhere," but now I see that this just gets replaced.  

Think it would make it a little clearer to rename to "The Base Journal Whose Name Gets Replaced" or something like that? Not sure, just thinking out loud.

Reply:
> "oh, okay, base journal must be in the test somewhere," but now I see that this just gets replaced.

The first thought was correct, no replacement. Updated because were were getting duplicate results because there is already a search result for "The New England journal of medicine" title in the test data: https://github.com/thirdiron/browzine-discovery-service-adapters/blob/develop/tests/unit/models/360-core.js#L19